docs/metrics.md

```diff
@@ -163,294 +117,51 @@ These are the metrics we plan to collect regarding the state of user datastores.
 
-### Events Related to Submitting An Item Change from the Item Editors
-Events of **category** `lockbox_item_change_submitted` record the actual submission of new item information to the datastore (either completely new entries or updates to existing items).
+7. `itemSelected` fires when a user clicks an item in the itemlist. **objects** itemList  
 
+8. `itemCopied` fires when a user copies their username or password from an item. **objects**: entryDetails
 
-In theory, these should be tied to equivalent events in the datastore, so that (for example) an `adding` event which is logged when a user submits a new item has a corresponding `added` event that gets logged when the item is successfully added to the datastore. This may need to change, but for simplicity's sake I've made it as close to the datastore events as possible (see next section). These events will be recorded as `lockbox_item_change_submitted`.
+9. `feedbackClick` fires when the user clicks the "Send Feedback" button. **objects**: manage
 
-**Methods** for this event are `adding`, `updating` and `deleting`.
+10. `resetRequested` fires when the user clicks the "Reset" button in the Lockbox settings. **objects**: settings
 
-The **object** can be `new_item` or `edit_item`.
+11. `resetCompleted` fires when the user completes a reset of their Lockbox data in the Lockbox settings. **objects**: settings
 
-The **extra** field contains the item's UUID *for updating and deleting events only*. For adding events, it always contains the string "new".
+## List of Planned Events
 
-To register these events, we would do something like:
-```javascript
-Services.telemetry.registerEvents("lockbox_item_change_submitted", {
-  "lockbox_item_change_submitted": {
-    methods: ["adding","updating","deleting"],
-    objects: ["new_item","edit_item"],
-    extra: {"fxauid": uid,
-            "item_id": UUID}
-  }
-});
-```
+These events will be implemented once the corresponding functionality is available.
 
-To record a new item submission (note the `item_id` is "new"):
-```javascript
-Services.telemetry.recordEvent("lockbox_item_change_submitted",
-                    "added",
-                    "new_item",
-                    null,
-                    {"fxauid": uid,
-                    "item_id": "new"});
-```
+1. `fxaSignIn` fires when someone clicks the signin button during firstrun. **objects**: fxaSignInPage.
 
-### Events Related to Changes in the Datastore
-We will record an event of **category** `lockbox_datastore.changed` when there is a change to the datastore. These events will be recorded as `lockbox_entry_changed`.
+2. `confirmPW` fires when the user clicks the button to confirm their FxA pw. **objects**: confirmPWButton
 
-**Methods** for this event are `added`, `updated` and `deleted`.
+3. `doorhangerItemSelected` fires when a user clicks an item in the doorhanger's itemlist. **objects** doorhanger  
```

Comment:
@jimporter in case you haven't seen it yet, here are the events that @irrationalagent has defined for the doorhanger #338